README.md

```diff
@@ -66,3 +66,14 @@ From architecture perspective, Orchard has a lower level API for port forwarding
 All port forwarding connections are done via the Orchard Controller instance which "proxies" a secure connection to the Orchard Workers.
 Therefore, your workers can be located under a stricter firewall that only allows connections to the Orchard Controller instance.
 Orchard Controller instance is secured by default and all API calls are authenticated and authorized.
+
+### Environment variables
+
+In addition to controlling the Orchard via the CLI arguments, there are environment variables that may be beneficial both when automating Orchard and in daily use:
+
+| Variable name                   | Description                                                                        |
+|---------------------------------|------------------------------------------------------------------------------------|
+| `ORCHARD_CONTEXT`               | Override the default context on per-command basis                                  |
```

Comment:
I don't thin there is a need for a variable that allows to "choose" context. If someone already configured a context via running CLI then they could've choose a context too. I think we can just remove such option and leave only environment variables for ovverriding.

Reply:
Removed in https://github.com/cirruslabs/orchard/pull/74/commits/0b06c0bb06391a371dbe5f097be3a556558bd941.